Added documentation for rebalance auto-approval (#7314)
strimzi/strimzi-kafka-operator: documentation/modules/cruise-control/proc-generating-optimization-proposals.adoc

  skipHardGoalCheck: true
----

> . Optionally, to have the optimization proposal to be automatically approved when it is ready, add the `strimzi.io/rebalance-auto-approval: true` annotation

Reviewer: ```suggestion
. (Optional) To approve the optimization proposal automatically, set the `strimzi.io/rebalance-auto-approval` annotation to `true`:
```
